### PR TITLE
Machine learning workflows updated

### PR DESCRIPTION
**Pull request by andrewssobral**

### CODE REVIEW

**Review by sophiesongge — APPROVED** [review]

**Review by sophiesongge — APPROVED** [review]

**Comment by sophiesongge** [comment]
retest this please

File: src/main/resources/default-workflows/Machine_Learning/Anomaly_Detection_Demo_WithAnomaly.xml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-    onTaskError="continueJobExecution"
-     maxNumberOfExecution="2"
->
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="Anomaly_Detection_Demo_WithAnomaly" projectName="Machine Learning"
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
>
```
Comment (marcocast) [inline src/main/resources/default-workflows/Machine_Learning/Anomaly_Detection_Demo_WithAnomaly.xml]: there is an extra pace i think

File: src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Image_Classification.xml [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
-    onTaskError="continueJobExecution"
-     maxNumberOfExecution="2"
->
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="TensorFlow_Demo_Image_Classification" projectName="Machine Learning"
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
>
```
Comment (marcocast) [inline src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Image_Classification.xml]: extra space i think

File: src/main/resources/default-workflows/Machine_Learning/Anomaly_Detection_Demo_WithAnomaly.xml (hunk 1 above)
Comment (sophiesongge) [reply src/main/resources/default-workflows/Machine_Learning/Anomaly_Detection_Demo_WithAnomaly.xml]: it's a return line

**Review by marcocast — APPROVED** [review]

**Pull request merged**